x-pack/plugins/session_view/public/components/ProcessTree/hooks.ts

```diff
@@ -121,9 +121,9 @@ export class ProcessImpl implements Process {
 
   isUserEntered() {
     const event = this.getDetails();
-    const { interactive, pgid, parent } = event?.process || {};
+    const { tty, group_leader: groupLeader, parent } = event.process;
 
-    return interactive && pgid !== parent.pgid;
+    return !!tty && groupLeader && groupLeader.pid !== parent.group_leader.pid;
```

Comment:
we can simplify this,   !!tty && process.pid === process.group_leader.pid

Reply:
Ahh alright, with that change I can revert the `parent: ProcessSelf`, since I thought we needed `parent.group_leader` here

Reply:
yea, we don't need to include those anemic nestings for now. they may have a use down the road, but we can keep simple and omit for now.